src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -61,13 +58,17 @@ export class ManifoldInvoices {
           (edge) => edge.node.id === this.selectedId
         );
 
+        if (!invoice) {
+          return null;
+        }
+
         return (
           <div>
             <button onClick={this.select()}>Back to all invoices</button>
             <h1 class="ManifoldInvoices__Subheading">
-              {new Intl.DateTimeFormat("en", {
-                year: "numeric",
-                month: "long",
+              {new Intl.DateTimeFormat('en', {
+                year: 'numeric',
+                month: 'long',
```

Comment:
The other changes in this file are just from running Prettier for the first time